src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+/* eslint-disable max-len */
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  selectedTabId : string;
+  tabs : Tab[];
+  onTabSelected : (tab : Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  selectedTabId,
+  tabs,
+  onTabSelected,
+}) => {
+  const selectTab = tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
+
+  const handleTabSelected = (tab : Tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              key={tab.id}
+              className={classNames({ 'is-active': tab.id === selectedTabId })}
```

Comment:
move it to the variable
```suggestion
              className={classNames({ 'is-active': ... })}
```